File: SoftUni_JAVA_BASIC_Jan_2023_SolutionsOfProblems/P02AdditionalProblems_ConditionalStatements/P01_PipesInPool.java
package P02AdditionalProblems_ConditionalStatements;

import java.util.Scanner;

public class P01_PipesInPool {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int v = Integer.parseInt(scanner.nextLine());
        int p1 = Integer.parseInt(scanner.nextLine());
        int p2 = Integer.parseInt(scanner.nextLine());
        double h = Double.parseDouble(scanner.nextLine());

        double vFull = h * (p1 + p2);
        if (vFull > v)
            System.out.printf("For %.2f hours the pool overflows with %.2f liters.", h, vFull - v);
        else
            System.out.printf("The pool is %.2f %% full. " +
                            "Pipe 1: %.2f %%. Pipe 2: %.2f %%.", (vFull / v) * 100,
                    ((h * p1) / vFull) * 100, ((h * p2) / vFull) * 100);
    }
}

File: JavaScript_Basic_04_2023_SolutionsOfProblems/02.MoreEx_JS_Conditional_Stitiments-More Exercises/P01_PipesInPool.js
function pipesInPool(input) {

    let volumePool = Number(input[0]);
    let flowPipe1 = Number(input[1]);
    let flowPipe2 = Number(input[2]);
    let timeHours = Number(input[3]);

    let volumeWather = (flowPipe1 + flowPipe2) * timeHours;

    if (volumeWather > volumePool) {
        console.log(`For ${timeHours} hours the pool overflows with ${(volumeWather - volumePool).toFixed(2)} liters.`);
    } else {
        console.log(`The pool is ${(100 * volumeWather / volumePool).toFixed(2)}% full. Pipe 1: ${(100 * flowPipe1 / (flowPipe1 + flowPipe2)).toFixed(2)}%. Pipe 2: ${(100 * flowPipe2 / (flowPipe1 + flowPipe2)).toFixed(2)}%.`);
    }
}
pipesInPool(["100",
    "100",
    "100",
    "2.5"
])